runtime/vm/extendedMessageNPE.cpp

```diff
@@ -214,6 +214,9 @@ convertMethodSignature(J9VMThread *vmThread, J9UTF8 *methodSig)
 			bufferSize += 3;
 			break;
 		case 'L': {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		case 'Q':
+#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
```

Comment:
While I think it's legal to put case labels within the `{ ... }` block, I don't think we should:
```C
		case 'L':
#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
		case 'Q':
#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
		{
```